internal/worker/standby.go

```diff
@@ -39,22 +30,21 @@ func (standby *StandbyConfig) UnmarshalYAML(value *yaml.Node) error {
 		return err
 	}
 
-	isolationNode := documentNode.FindChild("isolation")
-	if isolationNode == nil {
+	if isolationNode := documentNode.FindChild("isolation"); isolationNode == nil {
 		return ErrIsolationMissing
 	}
 	// Parse isolation
 	parserKit := &parserkit.ParserKit{
 		Boolevator:    boolevator.New(),
 		IssueRegistry: issue.NewRegistry(),
 	}
-	isolationParser := isolation.NewIsolation(nil, parserKit)
-	if err := isolationParser.Parse(isolationNode, parserKit); err != nil {
+	instanceParser := instance.NewStandbyParameters(nil, parserKit)
```

Comment:
Should this be renamed to avoid confusion? We don't parse an instance here, but standby parameters.

I think just naming it `parser` should do the trick.